### ISSUE TITLE
Composition vs Inheritance page is confusing as Inheritance is never talked about

### ISSUE THREAD

**Issue by jsunsawyer** [reactjs/reactjs.org#1950]
Would be nice to see an example or two of what you're comparing composition against, or at least an external reference to a use case in the context of React.

**Comment by kambleaa007** [comment]
can you provide more information about in which section or where the example would highly important

**Comment by jsunsawyer** [comment]
@kambleaa007 Here you go: https://reactjs.org/docs/composition-vs-inheritance.html

**Comment by kambleaa007** [comment]
Yeah, its true, redux documentation is quite need more improvements.
lots of things are uncovered or partially written

**Comment by kambleaa007** [comment]
What about containers, <Router>, <Provider> arent they Inherited.
Our whole <App /> wrapped inside them.
